src/validations/access-checker/items/collection-asset.ts

```diff
@@ -45,7 +45,7 @@ export type ItemCollection = {
 
 async function getCollectionItems(
   components: Pick<ContentValidatorComponents, 'externalCalls'>,
-  subgraphUrl: string,
+  subgraphUrl: ISubgraphComponent,
```

Comment:
nit: change `subgraphUrl` to something more meaningful such as `subgraphQuerier|subgraph|whatever`?